### PR TITLE
Small fixes on Network Dashboard cloud function code

### PR DESCRIPTION
**Pull request by simonebruzzechesse**
decreased severity of log in discover cai from INFO to DEBUG
added continue in case quota is missing for asset, otherwise runtime exception will raise for splitting self_link (None) value on line 49

### CODE REVIEW

File: blueprints/cloud-operations/network-dashboard/src/plugins/discover-compute-quota.py [hunk 1]
```diff
@@ -45,6 +45,8 @@ def _handle_discovery(resources, response):
     self_link = part.get('selfLink')
     if not self_link:
       logging.warn('invalid quota response')
+    else:
```
Comment (juliocc) [inline blueprints/cloud-operations/network-dashboard/src/plugins/discover-compute-quota.py:48]: else?

Comment (simonebruzzechesse) [reply blueprints/cloud-operations/network-dashboard/src/plugins/discover-compute-quota.py]: Sorry, I was out of my mind. Fixed!

**Review by ludoo — APPROVED** [review]
thanks!

**Review by ludoo — APPROVED** [review]
thanks!

**Pull request merged**